.eslintrc

```diff
@@ -1,6 +1,6 @@
 {
   "parser": "babel-eslint",
-  "extends": ["eslint:recommended", "prettier"],
+  "extends": ["eslint:recommended", "prettier", "plugin:no-jquery/deprecated-3.4"],
```

Comment:
How about upgrading `jQuery` to version 3.5.1, so we can just use `plugin:no-jquery/deprecated` here?